src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+
+  --text-color: #000;
+  --text-hover-color: #00acdc;
+}
+
+body,
+div,
+p,
+h1,
+h2,
+h3,
+a,
+li,
+ul,
+nav {
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  margin: 0 50px;
+  display: flex;
+  padding: 10px 0;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav-order {
+  display: flex;
+  list-style: none;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: var(--text-color);
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
on the feature - try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)